### PR TITLE
Fix the layout and styles for heroes

### PR DESCRIPTION
**Pull request by jimmynotjim**
Capital Framework made a dramatic change in the way we build heros to
provide future options for responsive images. These changes broke the
existing hero in this repo.

## Removals

- Removed the old hero less file

## Changes

- Updated the hero template to match markup in CFv4 examples

## Testing

1. run `./frontend.sh`

## Screenshots

screen shot 2017-07-12 at 10 31 51 am

screen shot 2017-07-12 at 10 32 37 am

screen shot 2017-07-12 at 10 32 08 am

screen shot 2017-07-12 at 10 32 16 am

screen shot 2017-07-12 at 10 45 15 am

screen shot 2017-07-12 at 10 45 08 am

## Notes

- In order to test each hero type, you'll have to grab the static urls and replace them in the template or your browsers inspection window. Maybe we should find a way to pull images from the server instead of locally in the future.
- See https://github.com/cfpb/capital-framework/pull/469 for history of hero change.

## Todos

- Update the image and padding settings to be programmatically set and delivered via Django

## Checklist

* [x] PR has an informative and human-readable title
* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [x] Project documentation has been updated
* [x] Reviewers requested with the [Assignee tool](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/) :arrow_right:

### CODE REVIEW

**Comment by nataliafitzgerald** [comment]
@jimmynotjim - The screenshots look correct. I don't know that I have an easy way to view these by hero type in the way you described above but I'm sure @anselmbradford is all over that. Thanks!

File: cfgov/jinja2/v1/_includes/molecules/hero.html [hunk 1]
```diff
@@ -51,60 +51,98 @@
...
+{# TODO: Programatically set the images and padding via Django #}
+{% set img=image(value.image, 'original') %}
+
+{% if value.small_image %}
+    {% set sm_img=image(value.small_image, 'original') %}
+{% else %}
+    {% set sm_img = image(value.image, 'original') %}
+{% endif %}
+
+{% set bp_sm_padding = "41.4893617%" %}
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/molecules/hero.html]: Could we add a note about why these magic numbers are necessary? Does it have to do with the django note on line 55?

**Comment by anselmbradford** [comment]
Can these comments be removed?
 - https://github.com/cfpb/cfgov-refresh/blob/f40c12655e78cbd987006891b8a554a4c4624d08/cfgov/jinja2/v1/_includes/molecules/hero.html#L20-L25
 - https://github.com/cfpb/cfgov-refresh/blob/f40c12655e78cbd987006891b8a554a4c4624d08/cfgov/jinja2/v1/_includes/molecules/hero.html#L27
 - https://github.com/cfpb/cfgov-refresh/blob/f40c12655e78cbd987006891b8a554a4c4624d08/cfgov/jinja2/v1/_includes/molecules/hero.html#L39

File: cfgov/jinja2/v1/_includes/molecules/hero.html [hunk 2]
```diff
@@ -51,60 +51,98 @@
...
-                    {% set img=image(value.image, 'original') %}
-                    src="{{ img.url }}"
-                {% endif %}
-                 alt="">
+            <div class="m-hero_image"></div>
         </div>
-    {% endif %}
     </div>
+    <style>
+        .m-hero {
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/molecules/hero.html]: Same as above, a note here on why inline styling is used would be helpful.

**Comment by jimmynotjim** [comment]
@anselmbradford Added some context to the issues you addressed and removed the unused variable comments from the top of the file. Ready for final sign off.

**Comment by anselmbradford** [comment]
👍

**Pull request merged**